vuepress/docs/next/tutorials/consume/solr.md

```diff
@@ -3,81 +3,77 @@ sidebarDepth: 2
 ---
 
 # Solr Integration
-Solr is an enterprise search platform with full-text search, real-time indexing, dynamic clustering, database integration, NoSQL features, and rich document handling. Solr can be integrated into applications made on Entando and is especially useful in applications built for high availiability multitenant architecture.
+Solr is an enterprise search platform with full-text search, real-time indexing, dynamic clustering, database integration, and rich document handling. Solr can be integrated into applications built on Entando and is required for multitenant architecture.
 
-To employ Solr, the index and configuration files called the core, has to be created. This tutorial decribes the instructions for both the general and multitenant use cases. For a mulitenant application, a core has to be generated for the primary and each of the secondary tenants. For secondary tenants, see the [Multitenacy on Entando](./multitenancy-tutorial.md) tutorial.
+This tutorial describes the installation steps for Solr integration, including the generation of the core collection which consists of the index and configuration files. In multitenancy applications, a core collection must be generated for the primary or first tenant, and each of the secondary tenants. 
 
 ## Prerequisites
-* [A working instance of Entando](../../docs/getting-started/README.md). with the default Tomcat server image
+* [A working instance of Entando](../../docs/getting-started/README.md) based on the default Tomcat server image
 * Verify dependencies with the [Entando CLI](../../docs/getting-started/entando-cli.md): `ent check-env develop`
-* [Helm](https://helm.sh/docs/intro/install/) to handle Solr installation
+* [Helm](https://helm.sh/docs/intro/install/) to handle the Solr installation
 
 ## Install and Configure Solr 
  
-1. Install Solr Helm charts  
-```
+1. Install Solr Helm charts:  
+``` bash
 helm repo add apache-solr https://solr.apache.org/charts
 helm repo update
 ```  
-For more information, see the [Solr Getting Started page](https://solr.apache.org/guide/7_3/solr-tutorial.html).
-
-2. Install the Solr and Zookeeper CRDs. **Note:** creating the CRDs and installing the operators is done at the cluster level and will require cluster-level permissions.
-```
+2. Install the Solr and Zookeeper CRDs:  
+**Note:** creating the CRDs and installing the operators are done at the cluster level and will require cluster-level permissions.
```

Comment:
@jyunmitch Does it look odd to have the semicolon on the previous line and a note before the actual steps?